napalm/eos/eos.py

```diff
@@ -158,6 +159,19 @@ def _multiline_convert(config, start="banner login", end="EOF", depth=1):
 
         return ret
 
+    @staticmethod
+    def _mode_comment_convert(config):
+        ret = list(config)
+        d = 1
+        s, _ = next((idx, l) for idx, l in enumerate(ret) if isinstance(l, py23_compat.string_types)
+                    and re.match(r"^(\s+)?!!", l))
+        while re.match(r"^(\s+)?!!", config[s + d]):
+            d = d + 1
+        ret[s] = {'cmd': 'comment', 'input': "\n".join(map(lambda s: s.lstrip("! "), ret[s:s + d]))}
+        del ret[s + 1:s + d]
```

Comment:
I always find using very abbreviated names makes the code much harder to read. I have to reverse engineer what the variable is from code. As opposed to having a pretty good idea what the variable is just from its name (and then having it make sense in the context of the code).